cmake/platforms/OpenCV-Linux.cmake

```diff
@@ -1 +1,10 @@
 # empty
```

Comment:
Lets remove this comment as file is not "intentionally empty" anymore.

Reply:
Done